schema/schema.go

```diff
@@ -90,6 +90,10 @@ func (s Schema) ValidatePatchOperation(operation string, operationValue map[stri
 		var attr *CoreAttribute
 		var scimErr *errors.ScimError
 
+		if k == CommonAttributeID {
+			continue
+		}
+
```

Comment:
https://developer.okta.com/docs/reference/scim/scim-20/#update-a-specific-group-name is considered as an error without this logic.

Response as an error here.
https://github.com/elimity-com/scim/blob/7f9449eb6fe0b544af7a2e537c9bd30f9182c3ca/schema/schema.go#L106-L108